src/ssl.c

```diff
@@ -2411,6 +2419,18 @@ int wolfSSL_Rehandshake(WOLFSSL* ssl)
     return ret;
 }
 
+
+int wolfSSL_Rehandshake(WOLFSSL* ssl) {
+    WOLFSSL_ENTER("wolfSSL_Rehandshake()");
+    return wolfSSL_StartSecureRenegotiation(ssl, 0);
+}
+
+
+int wolfSSL_SecureResume(WOLFSSL* ssl) {
```

Comment:
Does it make sense to add this new API `wolfSSL_SecureResume` wrapper? Why not just encourage  use of `wolfSSL_StartSecureRenegotiation` directly?

Reply:
It is a valid thing to do per the RFC. In general, we discourage the use of secure renegotiation at all.